contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -391,7 +382,7 @@ contract LoanTokenLogicStandard is LoanTokenSettingsLowerAdmin {
 		uint256 totalSupply = _totalAssetSupply(0);
 		uint256 totalBorrow = totalAssetBorrow();
 		if (totalSupply > totalBorrow) {
-			return totalSupply.sub(totalBorrow);
+			return totalSupply - totalBorrow;
```

Comment:
Avoiding safemath... there must be a good reason to do it. Any explanation?

Reply:
`if (totalSupply > totalBorrow)`
guarantees there won't be overflow
using safe math here is redundant, just wasting gas